semconvgen/generator.go

```diff
@@ -162,13 +164,20 @@ func render(cfg config) error {
 
 	args := []string{
 		"run", "--rm",
-		"-v", fmt.Sprintf("%s:/data", tmpDir),
+		"-v", fmt.Sprintf("%s:/data:z", tmpDir),
 		cfg.containerImage,
 		"--yaml-root", path.Join("/data/input/semantic_conventions/", path.Base(cfg.inputPath)),
+		"--only", cfg.onlyType,
+	}
+	if cfg.onlyType != "" {
+		args = append(args, "--only", cfg.onlyType)
+	}
```

Comment:
```suggestion
	}
	if cfg.onlyType != "" {
		args = append(args, "--only", cfg.onlyType)
	}
```

This would always add `--only` and then sometimes add it again.

Reply:
ups, seems to be a left over from a test run :face_exhaling: